doc/source/rst.rst

```diff
@@ -2,14 +2,174 @@
 Understanding the RST
 =====================
 
-Integrating Image
-=================
+If you want to edit OTC documentation you need to edit it using restructuredtext. 
+The basics of RST will be covered in this documentation. 
 
+Integrating Images
+==================
+
+If you simply want to implement a picture you can use following syntax:
+
+    .. code-block:: 
+        
+        |imagexx|
+
+       .. |imagexx| image:: /_static/images/filename
+
+   .. note:: This is an easy way to implement pictures espacially in tables. Ideally the file path would be at the bottom of your file. 
+
+The more elegant way is to use figures with following syntax:
+
+    .. code-block:: 
+
+        .. figure:: /_static/images/filename
+
+            :alt: This is your title 
```

Comment:
"alt" is so to say a hidden title (visible on hover if at all). The most proper way is to simply include title in the text form inside of the macros body (https://sphinxguide.readthedocs.io/en/latest/fig-num/fig.html)